main.go

```diff
@@ -92,6 +97,7 @@ func main() {
 		r.SecurityGroups = ctx.StringSlice("security-group")
 		r.Subnets = ctx.StringSlice("subnet")
 		r.Environment = ctx.StringSlice("env")
+		r.Count = ctx.Int64("number-of-tasks")
```

Comment:
@lox sorry, but I missed this! New PR coming :(

Reply:
Eeep I should have caught that too, sorry! Too early in the morning here :D